src/MeetingsSDKAdapter/controls/ExitControl.js

```diff
@@ -0,0 +1,37 @@
+import {Observable, of} from 'rxjs';
+import {MeetingControlState} from '@webex/component-adapter-interfaces';
+import MeetingControl from './MeetingControl';
+
+/**
+ * Display options of a meeting control.
+ *
+ * @external MeetingControlDisplay
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
+ */
+
+export default class ExitControl extends MeetingControl {
+  /**
+   * Attempts to leave the meeting of the given meeting ID.
+   * If the user had left the meeting successfully, a stopped event is dispatched.
```

Comment:
this is the jsdoc of the adapter leaveMeeting method. Nothing here tells us that a stopped event is dispatched, and if this changes in the adapter, one would have to come here and update this doc, too.

It's better to just describe just what this function does: calls the adapter leaveMeeting method.